.github/workflows/release.yml

```diff
@@ -2,92 +2,41 @@ name: Release
 
 on:
   workflow_dispatch:
-    inputs:
-      version_bump:
-        description: 'A type of version bump'
-        default: 'patch'
-        required: true
-        type: choice
-        options:
-          - major
-          - minor
-          - patch
+
 
 jobs:
   release:
     name: Release new version
-    runs-on: ubuntu-20.04
-
-    permissions:
-      contents: write
-      pull-requests: write
-
-    if: github.ref == 'refs/heads/develop'
```

Comment:
Let's add next condition `if: github.ref_protected == true` to run action only for protected branch

Reply:
Agreed